src/status_im/utils/http.cljs

```diff
@@ -45,6 +45,32 @@
                    (fn [error]
                      (utils/show-popup "Error" (str error))))))))
 
+(defn- headers [response]
+  (let [entries (es6-iterator-seq (.entries (.-headers response)))]
+    (reduce #(assoc %1 (string/trim (string/lower-case (first %2))) (string/trim (second %2))) {} entries)))
+
+(defn raw-get
+  "Performs an HTTP GET request"
```

Comment:
could you change the description, because it's the same as `defn get`, what's the difference?